activerecord/test/cases/adapters/mysql2/mysql2_adapter_test.rb

```diff
@@ -69,6 +69,13 @@ def test_errors_for_bigint_fks_on_integer_pk_table
     @conn.exec_query("ALTER TABLE engines DROP COLUMN old_car_id")
   end
 
+  def test_exec_query_returns_an_empty_result
```

Comment:
This test case should be passed by all adapters, add it in generic test cases (e.g. in `test/cases/adapter_test.rb`, not in adapter specific test cases).

Reply:
How can I run specific test files using `./bin/test` script?!

Reply:
e.g. `bin/test test/cases/adapter_test.rb -n test_select_all_always_return_activerecord_result`

Reply:
Thanks